sidebars.js

```diff
@@ -14,18 +14,138 @@
 /** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
 const sidebars = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
-  tutorialSidebar: [{ type: "autogenerated", dirName: "." }],
+  docs: [
+    {
+      label: "Getting Started",
+      type: "category",
+      collapsed: false,
+      link: {
+        type: "doc",
+        id: "getting-started/README",
+      },
+      items: [
+        {
+          type: "autogenerated",
+          dirName: "getting-started",
+        },
+      ],
+    },
+    {
+      label: "Core Protocol",
+      type: "category",
+      link: {
+        type: "doc",
+        id: "core-protocol/README",
+      },
+      items: [
+        {
+          type: "autogenerated",
+          dirName: "core-protocol",
+        },
+      ],
+    },
+    {
+      label: "Tutorials",
+      type: "category",
+      link: {
+        type: "doc",
+        id: "tutorials/README",
+      },
+      items: [
+        {
+          type: "autogenerated",
+          dirName: "tutorials",
+        },
+      ],
+    },
+    {
+      label: "Tools",
+      type: "category",
+      items: [
+        "tools/dashboard",
+        "tools/hardhat-plugin",
+        "tools/rpc-provider",
+        "tools/subgraph",
+        "tools/unlock.js",
+        {
+          type: "category",
+          label: "Sign in with ethereum",
+          link: {
+            type: "doc",
+            id: "tools/sign-in-with-ethereum/README",
+          },
+          items: [
+            {
+              type: "autogenerated",
+              dirName: "tools/sign-in-with-ethereum",
+            },
+          ],
+        },
 
-  // But you can create a sidebar manually
-  /*
-  tutorialSidebar: [
+        {
+          type: "category",
+          label: "Checkout",
+          link: {
+            type: "doc",
+            id: "tools/checkout/README",
+          },
+          items: [
+            {
+              type: "autogenerated",
+              dirName: "tools/checkout",
+            },
+          ],
+        },
+        {
+          type: "category",
+          label: "Paywall",
+          link: {
+            type: "doc",
+            id: "tools/paywall/README",
+          },
+          items: [
+            {
+              type: "autogenerated",
+              dirName: "tools/paywall",
+            },
+          ],
+        },
+
+        {
+          type: "category",
+          label: "Locksmith",
+          link: {
+            type: "doc",
+            id: "tools/locksmith/README",
+          },
+          items: [
+            {
+              type: "link",
+              href: "/api/locksmith",
+              label: "API",
+            },
+            "tools/locksmith/authorization-strategy",
+            "tools/locksmith/metadata",
+            "tools/locksmith/webhooks",
+          ],
```

Comment:
@julien51 They are actually reorganized from here now instead of sidebar position which is confusing to use.